mobile/src/Student/screens/Game/BasicGamePlay/index.js

```diff
@@ -73,47 +86,90 @@ const GamePreview = ({ navigation, route, gameSession, team, teamMember }) => {
         })
     }
 
-    const showNewHint = () => {
-        if (hints.length == availableHints.length) {
-            return
-        }
-        setHints([...hints, availableHints[hints.length]])
-    }
-
-    const showAllHints = () => {
-        setShowTrickAnswersHint(true)
-        setHints(availableHints)
-    }
-
     const submitAnswer = (answer) => {
-        global.apiClient
-            .addTeamAnswer(
-                teamMember.id,
-                question.id,
-                answer.text,
-                answer.isSelected
-            )
-            .then((teamAnswer) => {
-                if (teamAnswer == null) {
-                    console.error("Failed to create team.")
-                    return
-                }
-                console.log(teamAnswer)
-            })
-            .catch((error) => {
-                console.error(error.message)
-            })
-        if (!gameSession.isAdvancedMode) {
-            navigateToNextScreen()
-        }
+        Alert.alert(
+            "Are you sure?",
+            "You will not be able to change your answer",
+            [
+                {
+                    text: "Cancel",
+                    style: "cancel",
+                },
+                {
+                    text: "OK",
+                    onPress: () => {
+                        global.apiClient
+                            .addTeamAnswer(
+                                teamMember.id,
+                                question.id,
+                                answer.text,
+                                answer.isSelected,
+                                answer.isChosen
+                            )
+                            .then((teamAnswer) => {
+                                if (teamAnswer == null) {
+                                    console.error(
+                                        "Failed to create team Answer."
+                                    )
+                                    return
```

Comment:
Maybe show an alert or toast or we should add a text to the UI for error handling. Otherwise UX won't be that great and hard to know what went wrong